spec/formatting.md

```diff
@@ -652,6 +652,11 @@ MUST be an empty string.
 Implementations MAY offer functionality for customizing this,
 such as by emitting XML-ish tags for each _markup_.
 
+_Attributes_ MUST NOT have any impact on message formatting,
+and MUST NOT be available in the formatted output.
+_Variables_ in _attribute_ _values_ are not resolved,
+and no errors are emitted for them.
```

Comment:
This might be too strict? We do need to account for syntax errors. I would probably allow the variable resolution error.

Perhaps:

```suggestion
_Attributes_ are reserved for future standardization.
Implementations MUST emit a syntax error for any 
_attribute_ that is not well-formed. 
Implementations SHOULD NOT evaluate the value of
an _attribute_, but MAY emit an _Unrevolved Variable error_ if
one is encountered.
_Attributes_ SHOULD be omitted from any _fallback values_.
No other errors are permitted.
```

Reply:
I don't think syntax errors need to be specifically called out here, as they would invalidate the parsing of the source as a _variable_ or an _attribute_, so we never even consider this.

Otherwise the strictness here is necessary to ensure that attributes do not at this time provide for any runtime visibility. Even checking for Unresolved Variables could be externally observable.

Reply:
I'm not sure "runtime visibility" is a problem. We haven't defined with attributes are (at least formally). We've reserved a syntax and said it doesn't do anything yet. Maybe less is more?

```suggestion
_Attributes_ are reserved for future standardization.
Other than checking for valid syntax, they SHOULD NOT 
affect the processing or output of a _message_.
```